system/doc/reference_manual/expressions.xml

```diff
@@ -836,6 +836,21 @@ number &lt; atom &lt; reference &lt; fun &lt; port &lt; pid &lt; tuple &lt; map
       depending on the size of the float because otherwise comparison of large
       floats and integers would lose their transitivity.</p>
 
+    <p>The term equivalence operators, <c>=:=</c> and <c>=/=</c>, return
+      whether two terms are indistinguishable. While the other operators
+      consider the same <em>numbers</em> equal even when their types differ
+      (<c>1 == 1.0</c> is true), the term equivalence operators return whether
+      there exists any function that can tell their arguments apart.</p>
+    <p>For example, while the terms <c>0.0</c> and <c>-0.0</c> represent the
+      exact same <em>number</em>, they yield different results when converted
+      to string form through <c>float_to_list/1</c>: when given the former it
+      returns a string without a sign, and when given the latter it returns a
+      string with a sign. Therefore <c>=:=</c> and <c>=/=</c> consider these
+      terms different. This is useful when reasoning about terms as opaque
+      values, for example in associative containers or memoized functions where
```

Comment:
I still think it becomes confusing here, maybe partly because I am familiar with the history.

Above the text talks about `0.0` and `-0.0` and below it talks about `1` vs. `1.0`.
They should not be in the same paragraph.

I suggest to paragraph break here, and add to the paragraph above that also `0 =/= 0.0` so `0 =/= 0.0 andalso 0.0 =/= -0.0 andaso -0.0 =/= 0` is `true`, that is: all these 3 zeros are different.
Then the paragraph below can start with "The equivalence operators are useful when reasoning ...".

I think we should not mix up the new distinction between `0.0` and `-0.0` with the old distinction between `0` and `0.0`.

The Note further down about the "bug" in the equivalence between `0.0` and `-=0.0` makes things clear.  I like that.